### PR TITLE
[HU] Rules

### PR DESCRIPTION
**Pull request by Spkz**
### Description

Translates "Rules" to Hungarian language.

### Status

Finished

### Changelog

- Create hu.md
- Apply Kurokami's review

### CODE REVIEW

File: wiki/Rules/hu.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+2.  **Tartsd a posztjaidat és a válaszaidat a korhatár nélküli közösséghez megfelelően.** Felnőtt tartalomnak semmi keresnivalója egy korhatár nélküli közösségben, ugyanakkor semmi sem állít meg ezeknek a megvitatása privát üzenetekben.
+3.  **Legyél produktív a kritikáddal, anélkül, hogy átfolyna személyes támadásba.** A kritika egy nagyszerű dolog, ha rendesen van kivitelezve, de ha csupán személyes támadást eredményezel a kifejtéseddel, akkor rosszul csinálod, és el kéne gondolkozz ezen.
+4.  **Tartsd a szálakat relevánsan, és ne élessz fel régi témákat, ha kihaltak a beszélgetések.** Ha a téma már évek óta halott, nem igazán releváns többé, hagyd azt eltűnni. Posztolj újat, vagy keress és nézd meg, hogy valaki nem-e válaszolta meg már a kérdésed előtte.
+5.  **Az avatarjaid és az aláírásaid maradjanak ésszerűek.** Ha animáltak, ne villogjanak túlságosan. Egyúttal, a fájlméretük legyen alacsony, hogy a lassabb internetcsatlakozással rendelkezők is kényelmesen tudják böngészni az oldalt. Ez ideálisan maximum fél megabyte (500 kilobyte), ahol lehetséges.
```
Comment (Shadowthegod) [inline wiki/Rules/hu.md]: "Ezen kívül" instead of "Egyúttal"? That word sounds weird at the start of a new sentence.

File: wiki/Rules/hu.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+6.  **Ne posztolj hirdetéseket, átverős oldalakat vagy bármi egyéb tolakodó tartalmat.** Az átverős vagy partner oldalak megosztása jó mókának tűnhet annak, aki posztolja, de aki rákattint annak már jóval kevésbé az, tehát ne csináld. Az automatikusan betöltődő és lejátszódó tartalmak is szintúgy beletartoznak ebbe a szabályba.
+7.  **Ismeretlen célú applikációk vagy eszközök linkelése nem engedélyezett.** Egyszerűen, ha bármi legalitási kérdés felmerül az említett applikációk kapcsán, akkor nemkívánatosak posztolásra.
+
+Beatmap Közzétételi Szabályok
+---------------------------------
+
+Ezek a szabályok szigorúan azokra a beatmap-ekre vonatkoznak amik a játékon belüli [Beatmap Közzétételi Rendszer](/wiki/Beatmap_Submission_System) használatával lettek közzétéve. Ezek a szálak az osu! kliens által automatikusan jönnek létre, amikor a [szerkesztő](/wiki/editor) segítségével a beatmap-ek publikálásra kerülnek.
```
Comment (Shadowthegod) [inline wiki/Rules/hu.md]: "közzétéve" to "nyilvánosak"? All I keep reading is közzé, közzé...

File: wiki/Rules/hu.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+Léteznek jobban részletezett szabályok is a beatmap-ek felépítésére, lásd: [Rangsorolási Követelmények](/wiki/Ranking_Criteria).
+
+1. **Győződj meg arról, hogy engedélyed van mindenre, amit a beatmap-edben használsz.** Ez magába foglalja a zenéket, videókat, ütéshangokat, grafikákat, vagy bármi egyéb tartalmat, ami nem a te munkád gyümölcse. Ha nem vagy benne biztos, hogy merre kéne az efféle tartalmakat keresned, nekünk van egy folyamatosan bővülő zenegyűjteményünk a [Kiemelt Előadó](https://osu.ppy.sh/beatmaps/artists/) részlegnél, ahol az összes zene 100%-os licenccel rendelkezik az _osu!_ használatához.
```
Comment (Shadowthegod) [inline wiki/Rules/hu.md]: Please, can we avoid using bad pronunciation at "licenccel"? The word is with "sz" so it should be "licenc-szel" to be correct.

File: wiki/Rules/hu.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+2.  **Maradj témánál a Beatmap szálakban.** Ezek a szálak/beszélgetések csupán az adott beatmap-ről szólnak, és semmi másról. Ha bármi olyan problémád van, ami nem a beatmap-hez kapcsolódik, posztold azt a helyénvaló fórumba.
+3.  **Ne plagizálj vagy próbáld ellopni mások munkáját.** Szintúgy ne töltsd fel és használd a másik munkáját az engedélyük nélkül (beleértve, de nem korlátozva, témákat és vendég nehézségeket).
```
Comment (Shadowthegod) [inline wiki/Rules/hu.md]: The "plagizálj" is a kinda unknown word for youngsters so I would avoid using that, also that and "ellopni" means the same anyway.

File: wiki/Rules/hu.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+Ha a közösség rossz oldalán találod magad, rossz dolgok fognak történni.
+
+Ezek a rossz dolgok lehetnek (*de nincsenek korlátozva*):
+
+* A játékon belüli beszélgetési lehetőség megvonása egy adott időre (ismertebb nevén némítás)
+* Bármilyen sértő tartalom eltávolítása a felhasználói oldaladról, profilodról vagy a feltöltött beatmap-jeidről
+* A felhasználói oldalad, illetve a profilod részleteinek jövőbeli szerkesztési lehetőségének a megvonása
+* A hivatalos bajnokságok bejutására - illetve részvételére való jogosultságod megvonása
+
+A némítások és egyéb áthágások időtartama **automatikusan** növekszik majdnem exponenciálisan minden újabb megszegéssel. A moderátorok nem befolyásolják az adott sértésért járó némítás időtartamát - többszörös szabálysértők igen hosszú időtartamú elvonásban részesülnek.
```
Comment (Shadowthegod) [inline wiki/Rules/hu.md]: Comma between "növekszik majdnem"?

File: wiki/Rules/hu.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+9.  **Nyelvspecifikus csatornákban, a domináns nyelvet kell beszélned (Angol az #english-ben, Magyar a #hungarian-ben, stb.) ha beszélgetésben veszel részt.** Abban a csatornában, ahol a nyelv nincs kifejezetten előírva (vagy a csatorna nevéből nem lehet kikövetkeztetni, például Magyar a `#hungarian`-ben és Holland a `#dutch`-ban), ott mindig Angol. A `#lobby` és a `#help` csatornákban bármely nyelv megengedett, nem úgy, mint az összes többiben. *A [moderátori csapatnak](/wiki/People/Global_Moderation_Team) nem kell követnie ezt a szabályt, amikor a munkájukat végzik.*
+
+Fórum Szabályzat
+----------------
+
+1.  **Csak értelmes, érdekes tartalmakat/témákat posztolj.** A fórum elsősorban megvitatásokra van, szóval győződj meg arról, hogy elég energiát szántál arra, hogy ne rabold a többiek idejét.
+2.  **Tartsd a posztjaidat és a válaszaidat a korhatár nélküli közösséghez megfelelően.** Felnőtt tartalomnak semmi keresnivalója egy korhatár nélküli közösségben, ugyanakkor semmi sem állít meg ezeknek a megvitatása privát üzenetekben.
```
Comment (Shadowthegod) [inline wiki/Rules/hu.md]: You repeated "korhatár nélküli közösségben" twice which shouldn't be a problem but this is a short paragraph and it kinda hurts to read it more than once. Honestly, I have no idea how we could avoid this (I'm tired anyway), maybe we can't.

**Comment by TPGPL** [comment]
Mind using other branches for contributing next time. Never use master branch for it

**Comment by Shadowthegod** [comment]
Yeah, I scolded him for doing that already.

**Comment by Spkz** [comment]
Yes, sorry. I'm going to do so next time.

After several proofreads and hours of discussion with @Shadowthegod , this is the final version and is ready.

**Comment by TPGPL** [comment]
If it's ok, then you should approve this pr

**Pull request merged**